src/hotspot/cpu/aarch64/vm_version_aarch64.cpp

```diff
@@ -407,6 +408,29 @@ void VM_Version::initialize() {
     UsePopCountInstruction = true;
   }
 
+  if (UseBranchProtection == nullptr || strcmp(UseBranchProtection, "none") == 0) {
+    _rop_protection = false;
+  } else if (strcmp(UseBranchProtection, "standard") == 0) {
+    _rop_protection = false;
+    // Enable PAC if this code has been built with branch-protection and the CPU/OS supports it.
+#ifdef __ARM_FEATURE_PAC_DEFAULT
+    if (_features & CPU_PACA) {
```

Comment:
Style nit: no implicit booleans - expand as "if ( A & B != 0)"

Reply:
Oh yuck, really? This is my punishment for not paying attention to the style guide dicussions.